src/coreclr/jit/lsrabuild.cpp

```diff
@@ -686,7 +686,7 @@ bool LinearScan::isContainableMemoryOp(GenTree* node)
 //    mask        - the mask (set) of registers.
 //    currentLoc  - the location at which they should be added
 //
```

Comment:
nit: Comment about returning the RefTypeKill refposition.

Reply:
Will add that as part of a follow-up.